development-tools/clinic.rst

```diff
@@ -2001,6 +2001,44 @@ The generated glue code looks like this:
 .. versionadded:: 3.13
 
 
+.. _clinic-howto-getter:
+
+How to generate a getter
+------------------------
+
+You can use the ``@getter`` directive to generate an "impl" function
+that defines a `getter <https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef>`_.
+
+This example --- taken from :cpy-file:`Modules/_io/bufferedio.c` ---
+shows the use of ``@getter`` in combination with
+the :ref:`@critical_section <clinic-howto-critical-sections>` directive
+(which achieves thread safety without causing deadlocks between threads)::
```

Comment:
You've shown me how to _use_ a `getter` here, but you haven't told me anywhere what a `getter` function actually _is_ or why I'd use it. I assume it's like a getter of a property, right? Could you go into that a little bit more? :)

You could maybe give a REPL example of how the generated C extension behaves at the Python level after it's been compiled?

Reply:
Hmm, There might be some issue with markup

```
/home/runner/work/devguide/devguide/development-tools/clinic.rst:2009: WARNING: c:func reference target not found: PyGetSetDef
/home/runner/work/devguide/devguide/development-tools/clinic.rst:2039: WARNING: py:meth reference target not found: property.getter
```